builds/jazz-build-module/sls-app/serverless-build-rules.yml

```diff
@@ -398,15 +399,21 @@ provider:
   vpc: # Optional VPC. But if you use VPC then both subproperties (securityGroupIds and subnetIds) are required
     securityGroupIds:
       sbr-type: "[aws-id]"
-      sbr-render: user-only
+      sbr-render: config-wins
```

Comment:
VPC support isn't available yet. `config-wins` should be reverted back to `user-only` & `sbr-mandatory` to false. Same with `subnetIds `

Reply:
I guess this applies only to internal. Removed.